test/odbc/test_pg_datatype_bpchar.cpp

```diff
@@ -0,0 +1,650 @@
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include "odbc_handler.h"
+#include "query_generator.h"
+
+using std::pair;
+
+const string TABLE_NAME = "master_dbo.bpchar_table_odbc_test";
+const string VIEW_NAME = "master_dbo.bpchar_view_odbc_test";
+const string DATATYPE = "sys.bpchar";
+const int NUM_COLS = 4;
+const string COL_NAMES[NUM_COLS] = {"pk", "nvarchar_1", "nvarchar_4000", "nvarchar_20"};
+const int COL_LENGTH[NUM_COLS] = {10, 1, 4000, 20};
```

Comment:
Clears up when you do 
`ASSERT_EQ(col_length, COL_LENGTH)`
```suggestion
const int COL_LENGTH_EXPECTED[NUM_COLS] = {10, 1, 4000, 20};
```
Elsewhere too